src/main/java/com/suse/saltstack/netapi/datatypes/cherrypy/Arguments.java

```diff
@@ -0,0 +1,36 @@
+package com.suse.saltstack.netapi.datatypes.cherrypy;
```

Comment:
I think Arguments can go into datatypes directly the cherrypy is just for the stats stuff since that seems to be part of cherrypy and the netapi just taps into that.